cmd/janus/init.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"strings"
+
+	statsd "gopkg.in/alexcesaro/statsd.v2"
+	redis "gopkg.in/redis.v3"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/bshuster-repo/logrus-logstash-hook"
+	"github.com/hellofresh/janus/config"
+	"github.com/hellofresh/janus/middleware"
+)
+
+var (
+	err          error
+	globalConfig *config.Specification
+	accessor     *middleware.DatabaseAccessor
+	redisStorage *redis.Client
+	statsdClient *statsd.Client
+)
+
+// initializes the global configuration
+func init() {
+	globalConfig, err = config.LoadEnv()
+	if nil != err {
+		log.Panic(err.Error())
+	}
+}
+
+// initializes the basic configuration for the log wrapper
+func init() {
+	level, err := log.ParseLevel(strings.ToLower(globalConfig.LogLevel))
+	if err != nil {
+		log.Error("Error getting level", err)
+	}
+
+	log.SetLevel(level)
+	log.SetFormatter(&logrus_logstash.LogstashFormatter{
+		Type: globalConfig.Application.Name,
+	})
+}
+
+// initDatabase initializes a DB connection
+func init() {
+	accessor, err = middleware.InitDB(globalConfig.Database.DSN)
+	if err != nil {
+		log.Fatalf("Couldn't connect to the mongodb database: %s", err.Error())
+	}
+}
+
+// initRedis initializes a Redis connection
+func init() {
+	dsn := globalConfig.StorageDSN
+	log.Debugf("Trying to connect to redis instance: %s", dsn)
+	redisStorage = redis.NewClient(&redis.Options{
+		Addr: dsn,
+	})
+}
+
+// initStatsd initializes new StatsD client if it enabled
+func init() {
```

Comment:
Yes, that's how go works with init functions. You can have multiple init function in the same package, they will be executed one after the other in the order that they show up.